io.edgehog.devicemanager.Battery.json

```diff
@@ -0,0 +1,34 @@
+{
+  "interface_name": "io.edgehog.devicemanager.Battery",
```

Comment:
@rbino I'm not sure if we should call this `Battery` or with some kind of collective name.

Reply:
maybe BatteryStatus?